src/gleam/bit_array.gleam

```diff
@@ -155,3 +156,54 @@ pub fn base16_encode(input: BitArray) -> String
 @external(erlang, "gleam_stdlib", "base16_decode")
 @external(javascript, "../gleam_stdlib.mjs", "base16_decode")
 pub fn base16_decode(input: String) -> Result(BitArray, Nil)
+
+/// Converts a bit array to a string containing the decimal value of each byte.
+///
+/// ## Examples
+///
+/// ```gleam
+/// inspect(<<0, 20, 0x20, 255>>)
+/// // -> "<<0, 20, 32, 255>>"
+/// ```
+/// 
+pub fn inspect(input: BitArray) -> String {
+  "<<"
+  <> input
+  |> do_inspect(int.to_string)
+  |> string.join(", ")
+  <> ">>"
+}
+
+/// Converts a bit array to a string containing the hexadecimal value of each
+/// byte.
+///
+/// ## Examples
+///
+/// ```gleam
+/// inspect(<<0, 20, 0x20, 255>>)
+/// // -> "<<00 14 20 FF>>"
+/// ```
+/// 
+pub fn inspect_base16(input: BitArray) -> String {
```

Comment:
Could you remove this function please? It returns invalid syntax.

Reply:
Have removed it. Would it be worth including if it output valid syntax, i.e. `<<0x00, 0x14, 0x20, 0xFF, 0x8:5>>` ?